### ISSUE TITLE
Two fingers tap cause problem: Failed assertion: line 435 pos 12: '_initialButtons == kPrimaryButton': is not true.

### ISSUE THREAD

**Issue by 0100101101001011** [serenader2014/flutter_carousel_slider#215]
If you tap one of the CarouselSlider containers with more than one finger, the CarouselSlider freezes. The console produces the following error:

```
════════ Exception caught by gesture library ═══════════════════════════════════════════════════════
'package:flutter/src/gestures/monodrag.dart': Failed assertion: line 435 pos 12: '_initialButtons == kPrimaryButton': is not true.
```

**Comment by madc** [comment]
Can confirm this issue, we see the same behavior.

**Comment by dumuller** [comment]
I have the same problem

**Comment by serenader2014** [comment]
Fixed in v2.3.2, please upgrade to the latest version.
